include/aikido/planner/SequenceMetaPlanner.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef AIKIDO_PLANNER_SEQUENCEMETAPLANNER_HPP_
+#define AIKIDO_PLANNER_SEQUENCEMETAPLANNER_HPP_
+
+#include "aikido/planner/CompositePlanner.hpp"
+
+namespace aikido {
+namespace planner {
+
+class SequenceMetaPlanner : public CompositePlanner
```

Comment:
Class docstring? Is this for using a list of planners to solve one problem?

Reply:
This class is to solve multiple problems. If we want to have a composite planner to solve only one problem, then we could have
* `SingleProblemCompositePlanner<ProblemType>`
* `MultiProblemCompositePlanner` that is renamed from `CompositePlanner`.